data_specification/data_specification_generator.py

```diff
@@ -200,7 +203,7 @@ def reserve_memory_region(self, region, size, label=None, empty=False):
                     (Commands.RESERVE.value << 20) |  # @UndefinedVariable
                     (constants.NO_REGS << 16) |
                     (unfilled << 7) |
-                    (1 << 6) |
+                    (shrink << 6) |
```

Comment:
Maybe best to do `(int(bool(shrink)) << 6) |` here; enforces boolean-ness at the point where it _needs_ to be correct.